Denis_Keldanovich/06.AnsibleWorkshop/html/index.html

```diff
@@ -0,0 +1,8 @@
+<html>
```

Comment:
Please make as template, it should be on file, IP and host names will be variables